src/components/NibrsHistogram.js

```diff
@@ -29,23 +26,25 @@ class NibrsHistogram extends React.Component {
     const height = size.height - margin.top - margin.bottom
     const width = size.width - margin.left - margin.right
 
-    const bins = data.map(d => ({
+    const dataFiltered = data.filter(d => +d.key % 10 === 0)
+    const bins = dataFiltered.map(d => ({
       x0: +d.key,
       x1: +d.key + 10,
       ct: +d.count,
     }))
+
     const binCt = bins.length
+    const maxVal = max(bins, d => d.ct)
 
     const x = scaleLinear()
         .domain([min(bins, d => d.x0), max(bins, d => d.x1)])
         .range([0, width])
 
     const y = scaleLinear()
-        .domain([0, max(bins, d => d.ct)])
+        .domain([0, maxVal])
         .range([height, 0])
 
-    // default to first bin if no interaction
-    const active = hover !== null ? hover : 0
+    const active = hover || bins.filter(d => d.ct === maxVal)[0]
```

Comment:
This is fine, but instead of `filter()[0]` we can use `.find()` and it returns a single item and not an array.